templates/CRM/Member/Page/UserDashboard.tpl

```diff
@@ -24,7 +24,7 @@
             <th></th>
         </tr>
         {foreach from=$activeMembers item=activeMember}
-        <tr class="{cycle values="odd-row,even-row"} {$activeMember.class}">
+        <tr id="rowid{$activeMember.id}" class="{cycle values="odd-row,even-row"} {$activeMember.class}">
```

Comment:
@pradpnayak might want to use *membership_id_{$activeMember.id}* or *row_{$activeMember.id}* instead?